openpype/hosts/houdini/plugins/publish/validate_abc_path_attr.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+import pyblish.api
+
+
+from openpype.pipeline import PublishValidationError
+from openpype.hosts.houdini.api.action import FixPointCachePath
+from openpype.hosts.houdini.api.lib import PointCachePathHelper
+
+
+class ValidateAbcPathAttr(pyblish.api.InstancePlugin):
+    """Validate Alembic Path attribute is correct.
+    Path attribute should be equal to "{parent_node_name}/{output_node_name}"
+    """
+
+    order = pyblish.api.ValidatorOrder + 0.1
+    families = ["pointcache"]
+    hosts = ["houdini"]
+    label = "Validate Path Attribute (Abc)"
+    actions = [FixPointCachePath]
```

Comment:
you can take reference from other validators and use the Repair Action instead of your own implementation.

```suggestion
    actions = [RepairAction]
```
Make sure you import 
`from openpype.pipeline.publish import RepairAction`
By doing this,  you can implement repair function underneath the process function
You can take reference from the codes in other host if you are confused.
https://github.com/ynput/OpenPype/blob/53f5fc6d85cbb5b58cc624f58c760af628037fb3/openpype/hosts/max/plugins/publish/validate_frame_range.py#L61-L64C18